src/ml-engine/main_app/infrastructure/market_data.py

```
@@ -0,0 +1,77 @@
+from main_app.infrastructure.defi_llama import get_historic_tvl_and_apy_from_pool_id, get_historical_prices, get_pool_summary_data
+import json
+import pandas as pd
+from typing import List, Dict
+from datetime import date
```

Ruff (0.11.9)

`datetime.date` imported but unused

Remove unused import: `datetime.date`

(F401)